op-node/rollup/derive/channel_bank.go

```diff
@@ -83,61 +82,27 @@ func (ib *ChannelBank) IngestData(data []byte) {
 		panic("write data to bank while closed")
 	}
 	ib.log.Debug("channel bank got new data", "origin", ib.progress.Origin, "data_len", len(data))
-	if len(data) < 1 {
-		ib.log.Warn("data must be at least have a version byte, but got empty string")
-		return
-	}
-
-	if data[0] != DerivationVersion0 {
-		ib.log.Warn("unrecognized derivation version", "version", data)
-		return
-	}
-	buf := bytes.NewBuffer(data[1:])
 
+	// TODO: Why is the prune here?
 	ib.prune()
 
-	if buf.Len() < minimumFrameSize {
-		ib.log.Warn("data must be at least have one frame", "length", buf.Len())
+	// TODO: Technically breaking change that we require all frames to be parseable in a tx.data
+	frames, err := ParseFrames(data)
```

Comment:
Hmm this has implications for the batcher, I think that this is ok though. It seems like this requirement could make optimizing the batcher more difficult but I feel like it could greatly simplify code for handling frames